### PR TITLE
feat: Add higher quality pyhf logo

### PR DESCRIPTION
**Pull request by matthewfeickert**
* Use SVG converted PNG from Sticker Mule.

### CODE REVIEW

File: assets/images/projlogo/logo_pyhf.png
Comment (matthewfeickert) [inline assets/images/projlogo/logo_pyhf.png:1]: I think that this image is going to be too big dimensions wise. Do we have a standard size that we should aim for when adding/updating logos?

Comment (eduardo-rodrigues) [reply assets/images/projlogo/logo_pyhf.png:1]: Hi @matthewfeickert, I don't think we ever discussed such a point. In the "worst-case scenario" you could merge this in, check load times a few times for relevant pages, and decide if OK or not.

Comment (matthewfeickert) [reply assets/images/projlogo/logo_pyhf.png:1]: Sounds good to me. @henryiii unless you know off the top of your head we'll merge this tomorrow and check.

**Comment by matthewfeickert** [comment]
Looks fine in terms of size. :+1: Though it seems the pyhf logo is particularly terrible when it comes to dark mode rendering. :grimacing: @henryiii do you have any general advice for how to fix this when it comes to adding buffers around text to make it render well in both light and dark modes?

**Pull request merged**